hydrolib/core/dflowfm/polyfile/models.py

```diff
@@ -144,15 +144,18 @@ def y(self) -> List[float]:
         """Y-coordinates of all points in the PolyFile."""
         return [point.y for obj in self.objects for point in obj.points]
 
-    def get_z_sources_sinks(self) -> Tuple[float, List[float]]:
+    def get_z_sources_sinks(self) -> Tuple[List[float]]:
         """
         Get the z values of the source and sink points from the polyline file.
 
         Returns:
-            z_source, z_sinkA: Tuple[float, List[float]]:
+            z_source, z_sinkA: Tuple[List[float]]:
```

Comment:
I believe this now (wrongly) annotates a tuple with only 1 element (which is itself a list of floats).
The correct annotation is either `Tuple[List[float], List[float]]`, or a bit more freely:
`Tuple[List[float], ...]`
Quoting https://docs.python.org/3/library/typing.html#annotating-tuples :
> To denote a tuple which could be of any length, and in which all elements are of the same type `T`, use `tuple[T, ...]`

(also fix line 147 above)

Reply:
corrected